wcfsetup/install/files/ts/WoltLabSuite/Core/Form/Builder/Field/Button.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Data handler for a button form builder field in an Ajax form.
+ *
+ * @author  Matthias Schmidt
+ * @copyright 2001-2020 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Form/Builder/Field/Value
+ * @since 5.4
+ */
+
+import Field from "./Field";
+import { FormBuilderData } from "../Data";
+
+export default class Button extends Field {
```

Comment:
```suggestion
export class Button extends Field {
```
and add: `export default Button` at the bottom.